### DISCUSSION TITLE
Loading mesh with io.obj.import_mesh returns torch tensor with device set to 'cpu' by default

### DISCUSSION THREAD

**Discussion by karthikey97** [NVIDIAGameWorks/kaolin#682] [Q&A]
So I tried loading a .obj mesh file using kaolin.io.obj.import_mesh function. By default the tensors are created in the cpu. I cannot change the device to gpu in place because the function returns a namedtuple which is immutable. I went through the obj loading code and there does not seem to be any flexibility to specify which device the tensors get loaded to. Am I doing something wrong with my setup that tensors are not created in gpu by default? I am working off 0.13 installed using pip.

**Comment by shumash** [comment]
Please try with the latest version. The obj importer now returns a `SurfaceMesh` object, and you can call `cuda()` on it. 

See [tutorial docs here](https://kaolin.readthedocs.io/en/latest/notes/surface_meshes.html )
